src/lib/Strategy/Storage/GroupItemStrategyDispatcher.php

```diff
@@ -35,7 +35,10 @@ public function getGroupList(
             : $this->strategies;
 
         if (!isset($strategies[$groupBy])) {
-            throw new UnsupportedGroupItemStrategy($groupBy);
+            throw new UnsupportedGroupItemStrategy(
+                $groupBy,
+                implode(', ', array_keys($strategies))
```

Comment:
I think you should implode the array internally, inside the exception `__construct`.